app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1024,6 +1020,7 @@ public boolean onChildClick(ExpandableListView parent, View v, int groupPosition
 
             @Override
             public boolean onGroupClick(ExpandableListView expandableListView, View view, int position, long l) {
+                resetStreamSearch();
```

Comment:
Move this line to https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/activities/ZulipActivity.java#L1055.